src/components/input/file/style.css

```diff
@@ -44,21 +54,18 @@
 }
 
 :host>label {
-	padding-top: 0.4rem;
-	display: flex;
-	width: 100%;
-	align-items: baseline;
-	justify-content: flex-start;
-	color: rgb(var(--smoothly-input-foreground));
-	cursor: inherit;
-
+	position: absolute;
+	left: .5rem;
+	top: .6rem;
+	opacity: 0.8;
+	transform-origin: top left;
+	transition: transform 100ms ease;
```

Comment:
these rules are the same as for the general input. Instead of repeating them here, can we put them in the input/shared.css file?

Reply:
the same goes for any other rules that are duplicated.

Reply:
I'll see what I can do